TribalWars/src/ApplicationCore/Services/TribeService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using ApplicationCore.Entities;
+using ApplicationCore.Interfaces.Services;
+using ApplicationCore.Interfaces.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ApplicationCore.Services
+{
+    public class TribeService : ITribeService
+    {
+        public readonly ITribeRepository _repository;
+
+        public TribeService(ITribeRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public string CreateTribe(Tribe tribe, Guid playerId)
+        {
+            if (tribe != null && !TribeExist(tribe.Name))
+            {
+                TribePlayer tribePlayer = new TribePlayer();
+                tribePlayer.PlayerId = playerId;
+                tribePlayer.TribeRole = Enums.TribeRole.Owner;
+                tribePlayer.TribeId = _repository.CreateTribe(tribe);
+
+                if (tribePlayer.TribeId != 0)
+                {
+                    _repository.AddPlayerToTribe(tribePlayer);
+
+                    return "A tribe was created!";
+                }
+                return "Error while creating tribe!";
+            }
+            return "Tribe already exist!";
```

Comment:
exists* and I think the message could be more precise, like "The tribe's name is already taken/already exists"